install-dev/data/xml/configuration.xml

```diff
@@ -86,9 +86,6 @@
   <configuration id="PS_NB_DAYS_NEW_PRODUCT" name="PS_NB_DAYS_NEW_PRODUCT">
     <value>20</value>
   </configuration>
-  <configuration id="PS_SSL_ENABLED" name="PS_SSL_ENABLED">
-    <value>0</value>
```

Comment:
Doesn't you need to keep it but change it's value if you want to get the 1 as default one ?

Reply:
Thanks, that's a mistake. It is `PS_SSL_ENABLED_EVERYWHERE` which should have been removed but it is not present in this file.